pkg/utils/utils.go

```diff
@@ -7,6 +7,20 @@ import (
 	"github.com/openshift/origin/third_party/github.com/docker/libcompose/project"
 )
 
+// CheckIfFileExists checks if all files in array exists and that they are files (not directories).
+func CheckIfFileExists(files []string) error {
+	for _, filename := range files {
+		fileInfo, err := os.Stat(filename)
+		if err != nil {
+			return fmt.Errorf("main: file %q not found", filename)
```

Comment:
I mean that `CheckIfFileExists` will return error without `main:` in the error string and in `main` function in `henge.go` we do 
```go
fmt.Fprintf(os.Stderr, "main: %v\n", err.Error())
```
at https://github.com/redhat-developer/henge/pull/24/files#diff-837f59507063f9d7dfa76e57a60f456dL38